### ISSUE TITLE
Invalid nameserver address

### ISSUE THREAD

**Issue by hector2007ss** [nailfor/phpQueueClient#1]
I use the url: 10.10.1.81
but I get the error:
Fatal error: Uncaught InvalidArgumentException: Invalid nameserver address given in /var/www/html/examples/phpQueueClient-master/vendor/composer/../react/dns/src/Query/UdpTransportExecutor.php on line 108

**Comment by nailfor** [comment]
try use google DNS like this
```
        ClientFactory::run($url, $options, [$this, 'error'], new MQTT, '8.8.8.8');
```

**Comment by nailfor** [comment]
fixed in v1.2.7
